bind/gengo.go

```diff
@@ -111,6 +114,28 @@ func cgopy_decref(ptr unsafe.Pointer) {
 	refs.Unlock()
 }
 
+//export cgo_check
+func cgo_check() {
+	godebug := os.Getenv("GODEBUG")
+	cgocheck := -1
+	var err error
+	if godebug != "" {
+                for _, option := range strings.Split(godebug, ",") {
+                        if strings.HasPrefix(option, "cgocheck=") {
+                                cgocheck, err = strconv.Atoi(option[9:])
```

Comment:
instead of `option[9:]`, replace with:
`option[len(prefix):]`

where `prefix` is defined as:

```go
const prefix = "cgocheck="
```